src/schemas/validation/schema.yaml

```diff
@@ -760,8 +763,6 @@ $defs:
           properties:
             type:
               const: apiKey
-          required:
-            - type
```

Comment:
If `required` is unnecessary here, then why not also clean up https://github.com/karenetheridge/OpenAPI-Specification/blob/72de87b52b771209efcb4c1c89ed5358c2d66879/src/schemas/validation/schema.yaml#L960-L965?